AllJoyn/Platform/Bridge/BridgeCore/AllJoynHelper.cpp

```diff
@@ -1242,6 +1359,10 @@ QStatus AllJoynHelper::GetSignature(_In_ PropertyType propertyType, _Out_ std::s
         signature = "as";
         break;
 
+    case PropertyType::StringDictionary:
+        signature = "a{ss}";
```

Comment:
Ah ok. Good to know. thanks.

---
In reply to: [67958902](https://github.com/tjaffri/msiot-samples/pull/7#discussion_r67958902) [](ancestors = 67958902,67956680,67949524,67949261)